platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyUnresolvedModuleIntegrationTest.groovy

```diff
@@ -382,6 +464,11 @@ class DependencyUnresolvedModuleIntegrationTest extends AbstractHttpDependencyRe
         failure.assertHasCause("Read timed out")
     }
 
+    private void assertAdditionalFailuresMessagePrinted(int count) {
```

Comment:
This is only used in one place, can we just assert on that exact message above and drop all of the string interpolation here too?